src/main/resources/sql/insertDiscreteGroundWaterData.sql

```diff
@@ -1,64 +1,92 @@
 with upd as (
     insert into discrete_ground_water (
-                                       field_visit_identifier, location_identifier, start_time, end_time, party,
-                                       remarks, weather, is_valid_header_info, completed_work, last_modified, parameter,
-                                       parm_cd, monitoring_method, field_visit_value, unit, uncertainty, reading_type,
-                                       manufacturer, model, serial_number, field_visit_time, field_visit_comments,
-                                       publish, is_valid_readings, reference_point_unique_id,
-                                       use_location_datum_as_reference, reading_qualifier, reading_qualifiers,
-                                       ground_water_measurement
+                                       field_visit_identifier,
+                                       location_identifier,
+--                                        start_time,
+--                                        end_time,
+--                                        party,
+--                                        remarks,
+--                                        weather,
+--                                        is_valid_header_info,
+--                                        completed_work,
+--                                        last_modified,
+                                       parameter,
+                                       parm_cd,
+                                       monitoring_method,
+                                       field_visit_value,
+                                       unit,
+                                       uncertainty,
+                                       reading_type,
+--                                        manufacturer,
+                                       model,
+                                       serial_number,
+                                       field_visit_time,
+                                       field_visit_comments,
+                                       publish,
+--                                        is_valid_readings,
+                                       reference_point_unique_id,
+                                       use_location_datum_as_reference,
+--                                        reading_qualifier,
+                                       reading_qualifiers,
+--                                        ground_water_measurement,
```

Comment:
In general,  as a mater of style, commented out code should not be left in. Not familiar with what the IOW team
standards say. There are exceptions, such as perhaps in this case, it would be useful for when the missing meta data from
the original call is added to the new call by AQUARIUS, etc.  If the comments seem useful, fine leaving them in.

Reply:
@orosz-usgs yeah that was my reasoning for leaving these here as they are.  It's possible that in the near future we may be able to simply enable most of these fields again.

Reply:
@orosz-usgs I removed the comments from the select statements below, but left them in the insert portion to remind us which fields we still want from AQTS.